### ISSUE TITLE
ngc: empty metadata on windows

### ISSUE THREAD

**Issue by DevVersion** [angular/angular#15289]
**I'm submitting a**
* Bug Report

> Issue still appears after https://github.com/angular/angular/pull/15215 being merged

**Current behavior**

Metadata file when using `flatModuleOutFile` is empty on Windows 

```json
{"__symbolic":"module","version":3,"metadata":{},"importAs":"@angular/material"}
```

**Expected behavior**

Metadata file should contain `metadata` as for Linux and Mac OS.

```json
{"__symbolic":"module","version":3,"metadata":{"TestComponent":{"__symbolic":"class","decorators":[{"__symbolic":"call","expression":{"__symbolic":"reference","module":"@angular/core","name":"Component"},"arguments":[{"selector":"test-component","template":"<span>Test</span>"}]}],"members":{}}},"importAs":"@angular/material"}
```

**Minimal reproduction of the problem with instructions**

https://github.com/DevVersion/ng-compiler-cli-test

**Please tell us about your environment:**

Using the latest version (Github builds). And since `@angular/tsc-wrapped` is not bound to the `0.0.0-PLACEHOLDER`, there is a script that injects the latest version of it.

**Node (for AoT issues):** v6.10.0

cc. @jelbourn

**Comment by jelbourn** [comment]
cc @chuckjaz this is coming from one of the contributors for material

**Comment by IgorMinar** [comment]
related to #15192 ?

**Comment by chuckjaz** [comment]
This appears to be the same issue.

**Comment by chuckjaz** [comment]
Thanks for reporting this issue. However this issue is a duplicate of an existing issue #15192. Please subscribe to that issue for future updates.

**Comment by jelbourn** [comment]
@chuckjaz I asked @DevVersion to file this because he still encountered the issue after using the snapshot with the aforementioned fix.

**Comment by DevVersion** [comment]
Yeah, in #15192, the `metadata.json` files are not being emitted on Windows. 

And after using `tsc-wrapped-builds` with SHA https://github.com/angular/tsc-wrapped-builds/commit/6298778b57f40c892b6dc87742dad49e45627b64,  the `metadata.json` files are emitted.

Now those `metadata.json` files don't have any data included in the `{ metadata: EMPTY }` JSON.

I might be doing something wrong, but I specifically patched the `node_modules` with the `tsc-wrapped` version of `master`. ([see here](https://github.com/DevVersion/ng-compiler-cli-test/blob/master/package.json) & [here](https://github.com/DevVersion/ng-compiler-cli-test/blob/master/inject-tsc-wrapped.sh))

**Comment by chuckjaz** [comment]
Reopening issue as the fix for #15192 does not appear to fix this issue.
